srml/babe/src/lib.rs

```diff
@@ -241,6 +248,56 @@ impl<T: Trait> session::ShouldEndSession<T::BlockNumber> for Module<T> {
 	}
 }
 
+// TODO [slashing]: @marcio use this, remove the dead_code annotation.
+/// A BABE equivocation offence report.
+#[allow(dead_code)]
+struct BabeEquivocationOffence<FullIdentification> {
```

Comment:
What does equivocation in BABE look like?
- Authoring blocks on a chain that conflicts with the finalized chain.
- Authoring two different blocks in the same slot.
- Authoring for a slot that you weren't slot leader.

Reply:
According to the spec, an equivocation in BABE means that a validator produced more than one block in his slot.